packages/project-utils/packages/buildPackage.js

```diff
@@ -84,12 +84,16 @@ const babelCompile = async ({ cwd }) => {
 
     const writes = [];
     for (let i = 0; i < compilations.length; i++) {
-        const [file, { code, map }] = await compilations[i];
+        const [file, result] = await compilations[i];
+        const { code, map } = result;
 
         const paths = getDistFilePaths({ file, cwd });
         fs.mkdirSync(dirname(paths.code), { recursive: true });
         writes.push(fs.promises.writeFile(paths.code, code, "utf8"));
-        writes.push(paths.map, map, "utf8");
+        if (map) {
```

Comment:
Sure. Previously there was a bug that prevents from generating source maps at all, because `writes` array was supposed to be filled with promises, and instead variables for `writeFile` went there.
In case where maps are not enabled, `map` value is null, so we should not write it into file. In our setup it's always enabled, but user may want to use our `buildPackage` function with his own babel setup and disabled sourcemaps.

Maybe we should force source map generation at the level of generating packages, not `.babelrc` file? This way user will not be able to override that uninentionally. They won't miss anything on always generating source maps.

Reply:
Yeah I think that makes sense. Let's always have them, and not let the user worry about it. I guess then we could undo the change we did in `babelrc` file?